xml/System.Text.Json/Utf8JsonReader.xml

```diff
@@ -1596,13 +1715,13 @@ The comparison of the JSON token value in the source and the lookup text is done
 
 ## Remarks
 
-If the lookup text is invalid UTF-8 text, the method returns `false` since you can't have invalid UTF-8 within the JSON payload.
+The lookup text must be valid UTF-8 text. Otherwise, this method could return `true` if the source has a string token containing invalid UTF-8 text that matches.
```

Comment:
**Question:** Why do we have a preamble description/remarks section for the ValueTextEquals overloads?

https://docs.microsoft.com/en-us/dotnet/api/system.text.json.utf8jsonreader.valuetextequals?view=netcore-3.0#System_Text_Json_Utf8JsonReader_ValueTextEquals_System_String_

![image](https://user-images.githubusercontent.com/6527137/64501598-e5823380-d276-11e9-91e9-1506ecd4c52b.png)

Other overloads don't have that. Should we remove it? Specifically, the following:
https://github.com/dotnet/dotnet-api-docs/blob/1a26fca381874610894aa51a7c749dbbbdc94a6b/xml/System.Text.Json/Utf8JsonReader.xml#L1555-L1570

Reply:
It's because the overloads are wrapped inside a MemberGroup, to avoid repeating information.
The group has a general summary and a general remarks section that applies to all the overloads equally.
If you think the information is not relevant or accurate in the member group, feel free to alter it.

Reply:
Can I just remove it all together?

Reply:
Since the remarks are not all the same (and there are subtle differences in the summary too), I am removing the member group (especially in the interest of time since this doesn't affect intellisense anyway).

Reverting feedback from https://github.com/dotnet/dotnet-api-docs/pull/2890#discussion_r309999765

cc @mairaw